src/main/index.ts

```diff
@@ -146,9 +158,19 @@ if (!isFirstInstance) {
   };
 
   app.on('session-created', (session) => {
+    let reservedStateValues = [];
     if (!isDevelopment) {
       log.info('Cleaning local storage on session creation');
-      session.clearStorageData({ storages: ['localstorage'] }).catch(log.error);
+
+      if (mainWindow != null) {
+        mainWindow.webContents.executeJavaScript('({...localStorage});', true).then((localStorage) => {
```

Comment:
Since you have Promise chains, you should create an async function to make the code look cleaner.